### DISCUSSION TITLE
Is there a way to write sqcases syntax.

### DISCUSSION THREAD

**Discussion by SPY-LY** [KaTeX/KaTeX#3710] [Q&A] [answered]
I'm writing my conspectus in KaTeX and for specific reasons I need to write sqcases syntax, but I can't find a way to write it. So, is it supported some other way or will it be supported soon?

**Comment by edemaine** [comment, marked as answer]
Reading [the code for `cases`](https://github.com/KaTeX/KaTeX/blob/299584d12fa70c66fb9d5413a1503722b3a7e969/src/environments/array.js#L950), something like this works pretty well:

```latex
\def\arraystretch{1.2}
A=\left[\begin{array}{ll}
1 & \text{if } x=y\\
0 & \text{otherwise}
\end{array}\right.
```

![image](https://user-images.githubusercontent.com/2218736/190516533-f565f755-84dd-4dc9-ae25-0eef54e8ce5b.png)

(Maybe you also want a `\quad` between the columns...)

Assuming `sqcases` is in a LaTeX package (is it?), it might make sense to add support to KaTeX, and it should be pretty easy by following the linked code for `cases`.
